zookeeper-docs/src/main/resources/markdown/zookeeperUseCases.md

```diff
@@ -0,0 +1,319 @@
+    The configuration regarding all the topics including the list of existing topics, the number of partitions for each topic,
+    the location of all the replicas, list of configuration overrides for all topics and which node is the preferred leader, etc.
+    - **Access control lists**
+    Access control lists or ACLs for all the topics are also maintained within Zookeeper.
+    - **Membership of the cluster**
+    Zookeeper also maintains a list of all the brokers that are functioning at any given moment and are a part of the cluster.[9]
+
+### [Apache Mesos](http://mesos.apache.org/)
+  - Apache Mesos abstracts CPU, memory, storage, and other compute resources away from machines (physical or virtual),
+    enabling fault-tolerant and elastic distributed systems to easily be built and run effectively.
+  - Mesos has a high-availability mode that uses multiple Mesos masters: one active master (called the leader or leading master)
+    and several backups in case it fails. The masters elect the leader, with Apache ZooKeeper both coordinating the election
+    and handling leader detection by masters, agents, and scheduler drivers.[10]
+
+### [Apache Pulsar](https://pulsar.apache.org)
+  - Apache Pulsar is an open-source distributed pub-sub messaging system originally created at Yahoo and now part of the Apache Software Foundation
+  - Pulsar uses Apache Zookeeper for metadata storage, cluster configuration, and coordination. In a Pulsar instance:
+    - A configuration store quorum stores configuration for tenants, namespaces, and other entities that need to be globally consistent.
+    - Each cluster has its own local ZooKeeper ensemble that stores cluster-specific configuration and coordination such as ownership metadata,
+      broker load reports, BookKeeper ledger metadata, and more[24].
+
+### [Apache S4](https://github.com/apache/incubator-retired-s4)
+  - S4(**retired** on 2014-06-19[11]) is a general-purpose, distributed, scalable, partially fault-tolerant, pluggable platform that allows programmers
+    to easily develop applications for processing continuous unbounded streams of data.
+
+### [Apache Solr](https://lucene.apache.org/solr/)
+  - Solr is the popular, blazing-fast, open source enterprise search platform built on Apache Lucene.
+  - In the "Cloud" edition (v4.x and up) of enterprise search engine Apache Solr, ZooKeeper is used for configuration,
+    leader election and more[12,13]
+
+### [Apache Spark](https://spark.apache.org/)
+  - Apache Spark is a unified analytics engine for large-scale data processing.
+  - Utilizing ZooKeeper to provide leader election and some state storage, you can launch multiple Masters in your cluster connected to the same ZooKeeper instance.
+    One will be elected “leader” and the others will remain in standby mode. If the current leader dies, another Master will be elected,
+    recover the old Master’s state, and then resume scheduling[14]
+
+
+## Companies
+
+### 101tec
```

Comment:
i meant this entire section of "101tec" should be removed as it's already removed from wiki.